src/main/java/frc/robot/sensors/ArmSensor.java

```diff
@@ -0,0 +1,37 @@
+package frc.robot.sensors;
+
+import static edu.wpi.first.units.Units.Millimeter;
+
+import au.grapplerobotics.ConfigurationFailedException;
+import au.grapplerobotics.LaserCan;
+import edu.wpi.first.units.measure.Distance;
+import frc.robot.Hardware;
+
+public class ArmSensor {
+
+  private final LaserCan mainSensor;
```

Comment:
Recommend a more descriptive name for ease of debugging, e.g. mainLaserCanSensor or similar